website-team/proposals/zola.md

```diff
@@ -0,0 +1,43 @@
+# Zolaification aka One Website To Rule Them All
+The current website is mainly divided between three separate components, the main website, the blog, and "Thanks". This has caused a few issues since their release as we have continued to work on it, as the styling has to be duplicated on each, and any change has to be replicated on each or the styling will be out of sync.
+
+Not only is styling not shared between websites but each component is written in a unique Rust app, the main website is dynamic web server hosted on Heroku while the blog and thanks are separate bespoke static site generators that are deployed on GitHub Pages.
+
+This makes the barrier to contributing quite high. It would significantly reduce the maintenance burden if we can consolidate these websites into a single website, and we could move away from requiring nightly rust for building and running the website. Ideally we would use a static site generator, as we haven't had a need for bespoke backend for any features on the website.
+
+## Why Zola?
+
+There are a lot of different static site generators, so there's a good reason ask why pick one over another. Zola has quite a few unique features that make it particularly appealing to us.
+
+
+- **Independently maintained. —** We're not the sole or the biggest user of Zola, so there is plenty of community interest in continuing to use and add to Zola.
+- **Tera templating language —** Tera is Zola's backing templating language that is a much more fully featured templating language that allows us to write a lot of the logic that we have in Rust directly in the Tera templates themselves.
+- **Written In Rust —** Since Zola is written in Rust, it's much easier for Rust Web Presence members to contribute features to Zola that they need for the website. In fact some of that is currently in progress to add features currently missing from Zola that is present in our bespoke apps such as fluent support and multiple RSS feeds.
+- **Asset collocation —** Zola by default allows you to host static content from the same directory as the source markdown, this removes some of the absolute URLs that required on the blog because the static assets directory is seperate.
+
+
+## URL Scheme Changes
```

Comment:
This section does not talk about SEO implications of changing the URL scheme, nor how to mitigate the impacts.